server/package.json

```diff
@@ -6,15 +6,16 @@
   "main": "index.js",
   "scripts": {
     "start": "dotenv -e .env.pod -- node dist/server.js",
-    "build": "tsc && npm run sentry:sourcemaps",
+    "build": "tsc && npm run sentry:sourcemaps:inject",
     "dev": "MODE=dev dotenv -e .env.pod -- nodemon --watch src --ext ts --exec \"node --loader ts-node/esm\" src/server.ts",
     "dev:local": "dotenv -e .env.local -- nodemon --watch src --ext ts --exec \"node --loader ts-node/esm\" src/server.ts",
     "lint": "npx eslint src/ --ext .ts,.tsx",
     "migrate": "dotenv -e .env.pod -- npx prisma migrate dev --name ",
     "generate": "dotenv -e .env.pod -- npx prisma generate",
     "migrate:local": "dotenv -e .env.local -- npx prisma migrate dev --name ",
     "generate:local": "dotenv -e .env.local -- npx prisma generate",
-    "sentry:sourcemaps": "sentry-cli sourcemaps inject --org free-sleep --project server ./dist && sentry-cli sourcemaps upload --org free-sleep --project server ./dist"
+    "sentry:sourcemaps:inject": "sentry-cli sourcemaps inject --org free-sleep --project server ./dist",
+    "sentry:sourcemaps:upload": "sentry-cli sourcemaps upload --org free-sleep --project server ./dist"
```

Comment:
I've changed this to two separate scripts, since `sourcemaps upload` requires authentication, causing CI to fail. This also removes the error when building locally if you're not authenticated. In the future this could be added to a release workflow.